Create機能追加

sakura-on-the-blacklist opened the pull request:
タイトル、コンテンツを記入して、送信できるようになりました。
集められた情報はStore.phpにきちんと入るようになっています。
[https://i.gyazo.com/eaf1fb228b7cafc069b255fb95c5dce2.gif](url)

_index.php_のフォーマットは後で修正します。

sakura-on-the-blacklist committed:
Create機能追加・index.php修正

sakura-on-the-blacklist committed:
Create機能追加

sakura-on-the-blacklist committed:
Merge branch 'main' into create

MITSURUYAMAZAKI commented:
挙動確認しました。
<img width="683" alt="LGTM!!" src="https://user-images.githubusercontent.com/75934231/124471000-50e7ae80-ddd7-11eb-8ab5-f264fd3abb73.png">

chihiro0415 commented on store.php:
```diff
@@ -1,2 +1,17 @@
+//receive file
+$title = $_POST['title'];
+$contents = $_POST['contents'];
+$timestamp = time("Y年m月d日 H時i分s秒");
```
[MUST]
時間のフォーマットが違うので日時が登録できない状態です！
授業でやったように下記に変更してください 🙆‍♀️ 
```suggestion
$timestamp = date("Y/m/d H:i:s");
```

chihiro0415 commented on create.php:
```diff
@@ -6,13 +6,34 @@
-    
+
+    <link rel="stylesheet" href="style.css">
+    <style>
+        <?php include 'style.css';?>
+    </style>
```
[SHOULD]
これってなんで書いてますか...？？:eyes:

```
<link rel="stylesheet" href="style.css">
```
これと
```
<style>
    <?php include 'style.css';?>
</style>
```
これが同義なので、上だけで良いと思われます！ 🙆‍♀️

sakura-on-the-blacklist replied:
修正いたしました！
ご指摘ありがとうございます！

chihiro0415 commented on index.php:
```diff
@@ -1,32 +1,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Todoアプリ</title>
     <link rel="stylesheet" href="./style.css">
+    <style>
+        <?php include 'style.css'; ?>
+    </style>
```
createのコメント同様

sakura-on-the-blacklist committed:
不要な部分の削除

ご指摘ありがとうございます！
なぜ書いてあったのか謎です....　次回からは、不要なものが無いかしっかりチェックしながら、ケアレスミスを少なくしていこうと思います！

sakura-on-the-blacklist committed:
不要な部分の削除

sakura-on-the-blacklist committed:
時間の修正

chihiro0415 reviewed (approved):
![lgtm4](https://user-images.githubusercontent.com/30280895/124552641-c81c5180-de6e-11eb-8c50-7cdd099a2b2d.jpeg)
